attributes/default.rb

```diff
@@ -136,6 +136,15 @@
 default['cluster']['scheduler_plugin']['shared_dir'] = "#{node['cluster']['shared_dir']}/scheduler-plugin"
 default['cluster']['scheduler_plugin']['local_dir'] = "#{node['cluster']['base_dir']}/scheduler-plugin"
 default['cluster']['scheduler_plugin']['scheduler_plugin_substack_outputs_path'] = "#{node['cluster']['shared_dir']}/scheduler-plugin-substack-outputs.json"
+default['cluster']['scheduler_plugin']['virtualenv'] = 'scheduler_plugin_virtualenv'
+default['cluster']['scheduler_plugin']['pyenv_root'] = "#{node['cluster']['scheduler_plugin']['home']}/pyenv"
+default['cluster']['scheduler_plugin']['virtualenv_path'] = [
+  node['cluster']['scheduler_plugin']['pyenv_root'],
```

Comment:
cannot we reuse the pyenv installed in `prefix node['cluster']['system_pyenv_root']`?